src/components/Inputs/TextField/TextField.tsx

```diff
@@ -11,19 +11,19 @@ const StyledBase = styled(Base)`
   ${(props) => {
     if (!props.variant) {
       return `
-      input {
+      input, area {
```

Comment:
Is it intentional that this one is only `area,` while the other two are `textarea`